packages/react-ai/src/components/AIConversation/context/ControlsContext.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { ConversationInputContext } from './ConversationInputContext';
+import { SuggestedPrompt } from '../types';
+import { ConversationMessage } from '../../../types';
+
+export interface ControlsContextProps {
+  Form?: React.ComponentType<
+    {
+      handleSubmit: (e: React.FormEvent<HTMLFormElement>) => void;
+    } & Required<ConversationInputContext>
+  >;
+  MessageList?: React.ComponentType<{ messages: ConversationMessage[] }>;
+  PromptList?: React.ComponentType<{
+    suggestedPrompts?: SuggestedPrompt[];
+    setInput: ConversationInputContext['setInput'];
```

Comment:
Are we exposing this?

Reply:
Controls are exposed on the createAIConversation HOF similar to elements

Reply:
Got it. So we would be providing `setInput` to a consuming `PromptList` component right? Think we can probably remove it by massaging the `SuggestedPrompt` interface, can discuss